.eslintrc.json

```diff
@@ -0,0 +1,37 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": 2020,
+    "sourceType": "module",
+    "project": "./tsconfig.json"
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended",
```

Comment:
this checks typescript best practices
https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#supported-rules